README.md

```diff
@@ -33,14 +33,17 @@ For more details, see
   -  If you have issues, file them here: https://github.com/INCATools/ontology-development-kit/issues
   -  We also have an active community on Slack; you can request access by making a ticket [here](https://github.com/INCATools/ontology-development-kit/issues) as well
 
+# Steering Committee
+
+* @gouttegd Damien Goutte-Gattat (ODK Lead, Flybase)
```

Comment:
```suggestion
* @gouttegd Damien Goutte-Gattat (ODK Lead, FlyBase)
```

FlyBase is spelt with an uppercase B. This has slightly bothered me ever since my name had been added to that file, but I didn’t want to make an edit just for a letter. But since you’re making changes in that region… :D